ui/components/search/ReservationUnitCard.tsx

```diff
@@ -226,23 +220,38 @@ const ReservationUnitCard = ({
             ) : null}
           </Props>
           <Actions>
-            <div style={{ flexGrow: 1 }} />
             {containsReservationUnit(reservationUnit) ? (
               <MediumButton
-                iconLeft={<IconCheck />}
+                iconRight={<IconCheck />}
                 onClick={() => removeReservationUnit(reservationUnit)}
+                data-testid="reservation-unit-card__button--select"
```

Comment:
Both buttons have same testid?

Reply:
Well, they're semantically the same button.